test/codeu/chat/server/PermissionMapTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Tests the HashMap<ConversationHeader, HashMap<Uuid, byte>> way of 
+ * storing user permissions in Model. 
+ * 
+ * If you're seeing this, this is a tester class exploring storing
+ * an instantial set of user permissions. As of right now it's un-
+ * finished, but if anyone wants to trade for/take this task, or work
+ * with me on it, please let me know! ~ Sarah Abowitz 
+ * 
+ * @author (Sarah Abowitz) 
+ * @version (CA V1.0.4 | 7.15.17)
+ */
+
+package codeu.chat.server;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import codeu.chat.common.BasicController;
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+
+public class PermissionMapTest {
```

Comment:
Why use a separate class for these tests? Do you think you could just merge this with `ConversationServerAccessTest`? Both classes seem to be testing the same things.